### ISSUE TITLE
Cardview

### ISSUE THREAD

**Issue by Kap2022** [LemmyNet/lemmy-ui#757]
https://lemmy.ml/post/428464

A card view would make the Lemmy experience on a cell phone so much better and a lot more like the experience encountered on other sites.

At the moment on my cell phone, the pictures are too small. think how much better Memes would be if you could see the picture as you scroll through. But also I think it would generate a lot more user engagement overall.

If you do a cardview, make sure you put the headline under the picture. There has been a lot of studies done to look at how people read content. They focus on the picture and read the headline underneath the picture more frequently then they read the headline if it is above the picture.

**Comment by dessalines** [comment]
Will probably be a while before I get to this.

**Comment by Nutomic** [comment]
This would only make the experience better if the content consists mostly of images. I would consider Lemmy more like a forum, which is primarily based on text, so I wouldnt want such cards. I suppose it could be optional though.

**Comment by Kap2022** [comment]
I noticed Hexbear are porting some of their stuff up stream.

When you visit their site they have the images displayed larger then normal Lemmy.

**Comment by dessalines** [comment]
Yep I see this as being a radio switch with `(Card, List)` , maybe to the right of sorting. Although we're running low on real estate there.

**Comment by Kap2022** [comment]
That would work, but it would be good if the admin could set the default for the site - Card or List - in the Admin Settings.

The reason is visitors who are new to Lemmy might not know where the setting is and not hang around long enough to find out if it visually appealing to them.

**Comment by Cris-Edmundson** [comment]
Just wanted to voice I would also like this, since this thread hasn't been active for quite a while.

Thanks for all the work you do on lemmy!

**Comment by dessalines** [comment]
Should come along with #3428
